### PR TITLE
berlin design improvements, intro video

### PR DESCRIPTION
**Pull request by mei-li**
I hope its ok that we are adding a small video

### CODE REVIEW

**Comment by econchick** [comment]
Thanks for the email ping @mei-li !  if you ever need a review and I am not responsive, you can ping @willingc @estherbester and @audreyr as they can merge stuff too.

Video is a-okay in my PoV!  will deploy in a minute.

File: berlin/css/pyladies.css [hunk 1]
```diff
@@ -15,3 +21,55 @@ section.about_us, section.resources {
...
+  text-transform: uppercase;
+}
+
+video {
+  width: 100%;
+  cursor: pointer;
+  background-image: url("img/vidprelo.jpg");
+}
+video:hover {
+  background-image: url("img/vidpre.jpg");
```
Comment (econchick) [inline berlin/css/pyladies.css]: I think this and the background-image url a few lines above are giving a 404 - it looks like it should actually be `url("/img/vidpre.jpg")` (with the preceding `/`).
2015-05-28 at 2 51 pm

**Comment by econchick** [comment]
If you want, I can do that one minor change and deploy it; or you can fix, rebase, and force push to your branch.

**Comment by mei-li** [comment]
Change done, but I didn't rebase, no need from my side, except if it is a repo policy to keep a nice history

**Comment by econchick** [comment]
we do like to have a clean history - would you mind rebasing?

**Comment by mei-li** [comment]
sure no problem I rebased..

**Comment by econchick** [comment]
deployed - thanks so much!

**Pull request merged**